src/SFA.DAS.ApprenticeCommitments.Api/Controllers/ApprenticeshipController.cs

```diff
@@ -24,7 +25,7 @@ public ApprenticeshipController(IMediator mediator, IInternalApiClient<Apprentic
 
         [HttpPost]
         [Route("/apprenticeships")]
-        public async Task<IActionResult> AddApprenticeship(CreateApprenticeshipCommand request)
+        public async Task<IActionResult> AddApprenticeship(ApprenticeshipCreatedCommand request)
```

Comment:
This name sounds strange to me - it is both an event (`ApprenticeshipCreated`) *and* a command.

The Jobs function has received the `ApprenticeshipCreated` event.  Is it not, as a result, now giving APIM a `CreateApprenticeship` command?

Reply:
Agree, but the previous command CreateApprenticeshipCommand is also incorrect. We are not always creating an apprenticeship. We are actually responding to an approved apprenticeship and either creating a new apprentice (registration) or adding an apprenticeship to an existing apprentice